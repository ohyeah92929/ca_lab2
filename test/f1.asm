
.orig x3000
	add r0, r0, #2
	add r1, r0, #8
	lea r3, target
	ldb r4, r3, #0
	halt
target .fill x4321
.end
