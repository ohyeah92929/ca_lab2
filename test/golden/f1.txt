
Memory content [0x3000..0x3010] :
-------------------------------------
 0x3000 (12288) : 0x1022
 0x3002 (12290) : 0x1228
 0x3004 (12292) : 0xe602
 0x3006 (12294) : 0x28c0
 0x3008 (12296) : 0xf025
 0x300a (12298) : 0x4321
 0x300c (12300) : 0x0000
 0x300e (12302) : 0x0000
 0x3010 (12304) : 0x0000


Current register/bus values :
-------------------------------------
Instruction Count : 5
PC                : 0x0000
CCs: N = 0  Z = 0  P = 1
Registers:
0: 0x0002
1: 0x000a
2: 0x0000
3: 0x300a
4: 0x0021
5: 0x0000
6: 0x0000
7: 0x3008

